### ISSUE TITLE
If its not dead then need a fix

### ISSUE THREAD

**Issue by Az0na** [srhinos/new-openables#21]
Not opening items at all sins DragonFlight prepatch.

**Comment by Lisann76** [comment]
Hi srhinos, I have been using NOP for a few years now and have come to really appreciate it. I would be really happy if you could update it again for Dragonflight. Thank you very much in advance.

**Comment by srhinos** [comment]
Theres a very good chance I won't fix this, open to PRs with fixes but I stopped using this addon pretty quickly as the code is insanely unmaintainable and I just patched it to get it working good:tm: enough:tm:

**Comment by cont1nuity** [comment]
I have a working fork running with basic maintenance for DF. Will probably publish that on CF as well.
@srhinos: if you have any objections, please let me know. Happy to list you as co-author as well. :)

**Comment by srhinos** [comment]
yeah just go ahead and fork it, I don't intend on using this or contributing to it in any capacity as the source code (and performance) is horrid and manually adding items just drove me wild. If you ever take a crack at item detection without explicit item IDs or anything, happy to jump back in on this project as thats something I'd use!

**Comment by cont1nuity** [comment]
Already did a fork and maintenance release on CF/Wago yesterday :)
I had a look at automatic recognition, but it is a mess with the different global strings and snowflakes... Will certainly be improved when I find the time, but that's not on my prio right now. The global strings should exists localized and one should be able to parse them, but that's still a mess.

https://www.curseforge.com/wow/addons/new-openables-continued
https://addons.wago.io/addons/new-openables-continued

**Comment by Lisann76** [comment]
@cont1nuity Thank you so much for the effort to keep NOP alive! I installed it again immediately. :) Also, I took a good look at the German localization (I'm a native speaker) and made a few updates to it. Here is the link to the nop-locale-deDE.lua file: https://drive.google.com/file/d/1RwunKhCZWg13ytuL3Ys39HSmKRpsbqVk/view?usp=sharing

**Comment by cont1nuity** [comment]
@Lisann76 Thank you. It is included in the next alpha version.
I think we can safely close this ticket now. My fork for the projects mentioned above: https://github.com/cont1nuity/new-openables-continued can be used for further tickets.
